src/components/cohorts/CohortPage.js

```diff
@@ -1,27 +1,92 @@
 import Header from '../Header/Header';
 import client from '../../utils/client';
-import { useParams } from 'react-router-dom';
+import { useParams, useNavigate } from 'react-router-dom';
 import { useState, useEffect } from 'react';
-
+import './style.css';
 
 const CohortPage = () => {
   const params = useParams();
-  const [cohort, setCohort] = useState()
+  const navigate = useNavigate()
+  const [cohortName, setCohortName] = useState()
+  const [studentsInCohort, setStudentsInCohort] = useState()
+  const [availibleUsers, setAvailibleUsers] = useState()
 
   useEffect(() => {
     client
         .get(`/cohort/${params.id}`)
-        .then((res) => {
-          const cohort = res.data.data
-          setCohort(cohort)
-        })
+        .then((res) => setCohortName(res.data.data.cohortName))
```

Comment:
I'd expect to be able to return cohort data with a list of its students, rather than have to also make an additional request to get data for users by cohort.

Let's create an issue to refactor the endpoints and the client side requests.